src/Maestro/Maestro.Web/Startup.Api.cs

```diff
@@ -41,6 +41,8 @@ private void ConfigureApiServices(IServiceCollection services)
             services.AddSwaggerGen(
                 options =>
                 {
+                    options.CustomSchemaIds(x => x.FullName);
```

Comment:
Why did you add this?

Reply:
Without this I get an error saying that I have a duplicated type Build, because by default swagger doesn't consider namespaces on the type name. Error is this: `Identical schemaIds detected for types Maestro.Web.Api.v2019_01_16.Models.Build and Maestro.Web.Api.v2018_07_16.Models.Build`. See comment [here](https://stackoverflow.com/a/47306578/973795) also.

Reply:
This shouldn't be necessary, the old Build type should not be included in the new api. If it is thats a bug.

Reply:
Fix included in next push.